Resources/Prototypes/_DV/Entities/Objects/Fun/instruments.yml

```
@@ -25,12 +25,17 @@
     tags:
     - StringInstrument
   - type: MeleeWeapon
+    wideAnimationRotation: -45 # Added wideAnimationRotation
     soundHit:
       path: /Audio/_DV/Weapons/electricguitarhit.ogg
     damage:
       types:
-        Blunt: 7
-    bluntStaminaDamageFactor: 1.50 #13.5 stamina damage
+        Blunt: 8 # Increased by one
```

```suggestion
        Blunt: 8 # Increased by one
```
This is in _DV, no need for commenting changes
Also the antag version still only has 7 damage by default